psi4/driver/procrouting/dft/dh_functionals.py

```diff
@@ -618,25 +618,18 @@
 
 
 def get_pwpb95_tweaks():
-    beta = 5.0 * (36.0 * 3.141592653589793)**(-5.0 / 3.0)
     X2S = 0.1282782438530421943003109254455883701296
-    X_FACTOR_C = 0.9305257363491000250020102180716672510262  #    /* 3/8*cur(3/pi)*4^(2/3) */
     bt = 0.004440  # paper values
     c_pw = 0.32620  # paper values
     expo_pw6 = 3.7868  # paper values
     alpha_pw6 = c_pw / X2S / X2S
-    a_pw6 = 6.0 * bt / X2S
-    b_pw6 = 1.0 / X2S
-    c_pw6 = bt / (X_FACTOR_C * X2S * X2S)
-    d_pw6 = -(bt - beta) / (X_FACTOR_C * X2S * X2S)
-    f_pw6 = 1.0e-6 / (X_FACTOR_C * X2S**expo_pw6)
-    return ([a_pw6, b_pw6, c_pw6, d_pw6, f_pw6, alpha_pw6, expo_pw6])
+    return dict(zip(["_bt", "_alpha", "_expo"], [bt, alpha_pw6, expo_pw6]))
```

Comment:
Explicit `{"_bt": bt, ...}`? I haven't seen this form of dictionary initialization before.